
exports.triggerError = (req, res, next) => {
  next(new Error('ðŸš¨ Intentional 500 error for testing'));
};
